setup.py

```diff
@@ -264,6 +264,9 @@ def build_libraries(self, libraries):
     cmdclass={"build_clib": build_clib_hpy},
     use_scm_version=get_scm_config,
     setup_requires=['setuptools_scm'],
-    install_requires=['setuptools>=64.0'],
+    install_requires=[
+        'setuptools>=64.0',
```

Comment:
I have no idea, should I remove this?

Reply:
Does the code import either `setuptools` or `pkg_resources` anywhere in the code base after your changes? If not, you can remove the dependency.

Reply:
Frankly `setuptools` should never be a run-time dependency of **any** project. Its only legitimate use at the moment is as a build back-end.

Reply:
looks like `hpy.devel` still require setuptools

Reply:
in my understanding, HPY_UNIVERSAL package still need `hpy.universal` as runtime dependency, right?

maybe `importlib_resources` should be a runtime dependency of `hpy.universal` only

Reply:
Is that a separate project?

Reply:
Is `hpy.universal` a separate project or is it somehow packaged separately from this project?

Reply:
Anyway, if `setuptools` is directly imported by this project, it should be kept as a run-time dependency. At best it could be moved to extras if it's only used in some edge cases, provided that it's properly documented that way.

Reply:
> Is `hpy.universal` a separate project or is it somehow packaged separately from this project?

I don't know 🧐， didn't even successfully pack my hpy universal mode wheel yet.